TereseBo/Lingon
# Add mongoose

## Description

This PR conatins the functionality to connect the application to the database

## Changes Made

- [x] Added Mongoose and mongo db. Connection functionality, a trial mongoose model as well as back and front end for testing connection and vizualise DB data.
- [x] Refactored mentioned files 

## How to Test

1. Go to the page at /, click button to load DB data and render it in the page
2. Check if code for DB connection is clean and understandable, including comments

## Checklist
